kubeflow/core/prototypes/iap-envoy.jsonnet

```diff
@@ -4,7 +4,7 @@
 // @shortDescription Envoy proxies to handle ingress and IAP.
 // @param name string Name to give to each of the components
 // @optionalParam namespace string default Namespace
-// @optionalParam envoyImage string gcr.io/kubeflow/envoy:v20180124-0.4.0-50-g0d29aac-dirty-4d9e20 The image for envoy.
+// @optionalParam envoyImage string gcr.io/kubeflow-dev/envoy:0fb4886b463698702b6a08955045731903a18738 The image for envoy.
```

Comment:
nit: This is perfectly for now. But in general I suggest we follow a naming convention of something like the following for labels
vYYYYMMDD-GITHASH

Using the date is convenient for humans to ascertain at a glance which images are newer. The GITHASH does 2 thing
1. It ensures if we build a new image from new code we get a different tag 
2. It makes it easy to link an image to its source.

Reply:
ack